package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "app",
+  "packageManager": "yarn@3.3.0",
+  "scripts": {
+    "start": "npm install yarn -g --save && yarn set version berry && yarn",
```

Comment:
`start` se usa normalmente para iniciar una web; en este caso creo the `init` sería más correcto, aunque ese lo usa `npm` internamente; quizás `setup` o algo así.